app/assets/javascripts/compute_resource.js

```diff
@@ -66,7 +66,7 @@ function testConnection(item) {
 
 function ovirt_templateSelected(item){
   var template = $(item).val();
-  if (template && !item.disabled) {
+  if (!item.disabled) {
```

Comment:
With no template selected the default template which does have memory and cpu count will return in the result (in the ajax).

The other option is ignoring the blank template and in case of no template removing the values of all the fields in the virtual machine tab.